### ISSUE TITLE
Rocket transform Regression : instable prédictions

### ISSUE THREAD

**Issue by IKetchup** [sktime/sktime#4374]
I am using rocket transform and ridge regression to predict timeseries data.

```
x_train shape : (800, 644, 36)
x_test shape  : (200, 644, 36)
y_train shape : (800, 644 * 1008)
y_test shape  : (200, 644 * 1008)
```

I fit and transform my train and test data using rocket transform :

```
model = MiniRocketMultivariate(num_kernels=5000)
model.fit(x_train)

x_trf = model.transform(x_train)
x_test_trf = model.transform(x_test)
```

Then I train a ridge regression and use it to predict y_test :

```
ridge = Ridge(max_iter=30000, random_state=116)
ridge.fit(x_trf, y_train)
ridge_pred = ridge.predict(x_test_trf)
```

How ever when I print my prediction and compare it to the truth I notice that :
- Predictions globally follow the truth
- I have a LOT of noise

Here is an exmeple of my truth data and my predictions :

![bokeh_plot (34)](https://user-images.githubusercontent.com/72007646/226592138-b6ca1ddd-dab7-4a14-ad94-f99984bb2d6e.png)

I also tried with other regression model I have the same problem.

Any idea why I get so much noises ?

**Comment by haskarb** [comment]
Did you put seed for MiniRocket?

**Comment by IKetchup** [comment]
No

**Comment by haskarb** [comment]
You should try that. Also, MiniRocket could have issues with amplitude

**Comment by IKetchup** [comment]
I have the same problem if i use the random_state parameter. What do you mean by "MiniRocket could have issues with amplitude" ?

**Comment by haskarb** [comment]
If I remember correctly, MiniRocket normalises the signal. You can't do
much about it instead of re-writing MiniRocket code.

**Comment by ChildEater69** [comment]
Can you share the codes that you are woking on ,i want to see the initial codes of that code.
